### ISSUE TITLE
404 page not found on company delete

### ISSUE THREAD

**Issue by pira067** [grokability/snipe-it#14902]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

1. Whenever I try to edit/delete the company i get 404 page not found error
2. even after I created the company It's not showing in the asset creation

### Reproduction steps

1.Delete the company lead to error
2.
3.
...

### Expected behavior

it's suppose to delete the file and the 2nd issue suppose to show company when i am creating the assets

### Screenshots

![image](https://github.com/snipe/snipe-it/assets/173185034/09d092f6-393c-459b-91bf-bd72498c1384)
![image](https://github.com/snipe/snipe-it/assets/173185034/8ba7f49d-d9cf-4e17-815f-c02fb703f0f2)

### Snipe-IT Version

7.0.3

### Operating System

Windows

### Web Server

Apache

### PHP Version

8.2.12

### Operating System

Windows

### Browser

Chrome

### Version

126.0.6478.61

### Error messages

```shell
45
46
47
48
            return $this->getRouteForMethods($request, $others);
        }

        throw new NotFoundHttpException(sprintf(
            'The route %s could not be found.',
            $request->path()
        ));

8.2.12
1.03s
29MB
10.48.4
production
en-US
ERRORlaravel.log[2024-06-18 17:41:52] production.ERROR: Maximum execution time of 600 seconds exceeded {"exception":...
ERRORlaravel.log[2024-06-18 17:41:52] production.ERROR: Symfony\Component\ErrorHandler\Error\FatalError: Maximum exe...
```

### Additional context

Fresh Install
Manual installation
same error on other browsers as well

**Comment by snipe** [comment]
That definitely sounds odd. I can't recreate this on the demo or locally. Did anyone on your end perhaps change any files?

**Comment by pira067** [comment]
> That definitely sounds odd. I can't recreate this on the demo or locally. Did anyone on your end perhaps change any files?

Not at all. I just created new account for my organization and Followed the installation process. So it's just 1 day old account.

**Comment by snipe** [comment]
I am still unable to reproduce this, unfortunately :(

Can you run these commands and see if that improves anything?

```
git pull
php composer.phar install --no-dev --prefer-source
php composer.phar dump-autoload
php artisan migrate
php artisan config:clear
php artisan cache:clear
php artisan route:clear
```

**Comment by snipe** [comment]
@pira067  any update?

**Comment by snipe** [comment]
Hi there - We haven't heard back in a bit, so I'm going to close this ticket for now, but will re-open it if you're still having issues.

![hackerman-hacking](https://user-images.githubusercontent.com/197404/31936003-27a7f6d8-b865-11e7-8f5e-eb03062925f4.gif)
